lib/i18n-js/embed_fallback_translations_plugin.rb

```diff
@@ -16,33 +44,21 @@ def validate_schema
     end
 
     def transform(translations:)
-      translations_glob = Glob.new(translations)
-      translations_glob << "*"
+      return translations unless enabled?
 
-      mapping = translations.keys.each_with_object({}) do |locale, buffer|
-        buffer[locale] = Glob.new(translations[locale]).tap do |glob|
-          glob << "*"
-        end
-      end
-
-      default_locale = I18n.default_locale
-      default_locale_glob = mapping.delete(default_locale)
-      default_locale_paths = default_locale_glob.paths
+      fallback_locale = I18n.default_locale.to_sym
+      locales_to_fallback = translations.keys - fallback_locale
```

Comment:
Shouldn't this be `[fallback_locale]` instead? I'm running this in development and I'm getting an exception.

```suggestion
      locales_to_fallback = translations.keys - [fallback_locale]
```

Reply:
Sorry, my bad. It should be `translations.keys - [fallback_locale]` 👍